.github/workflows/ci.yaml

```diff
@@ -0,0 +1,61 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  phpunit:
+    name: "PHPUnit"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      fail-fast: false
+      matrix:
+        symfony-version:
+          - '^4.4'
+          - '^5.4'
+          - '^6.4'
+        php-version:
+          - "7.4"
```

Comment:
In composer.json you have PHP 7.2, it tests on PHP 7.4, but I would remove support for <8.0, and rewrite the code accordingly (eg. dump `@var` annotations everywhere, etc.); you can use rector for automatically fixing up the codebase.

Reply:
Also, Symfony version should be the lowest that the monolith is using, `4.4` should not be tested.